### PR TITLE
Update web updates

### PR DESCRIPTION
**Pull request by pshriwise**
This contains bunch of edits. I'm going to add a few questions here about other edits on various pages I was hesitant to make without more information. 

**DAGMC Installation Overview**

- link to some tutorials on unix/linux shell basics? (SWC maybe?)
- FLUKA or FLUKA source code? Is it necessary to be explicit about this like MCNP5? (honest question)

**FLUDAG**

 - All volumes must belong to a group… will this cause FluDAG to fail or will the volumes be empty as in DAG-MCNP? In which case, can we just ask them to create an empty group. I think this is less confusing than assigning a pseudo-material to the graveyard

**DAG-Tripoli** 

- Do you need to add a volume to the MCNP implicit compliment material group as well? (Same as FLUDAG question)

**Producing Solid Models** 

- Explain how we go about flooding a model maybe?

**DAGMC Unstructured Mesh Tally** 

- Some of this seems out of date. (e.g. mbconverting ACIS files to .h5m is no longer possible)

**DAGMC Tools**

- `make_watertight` image seems off...

### CODE REVIEW

**Comment by makeclean** [comment]
**link to some tutorials on unix/linux shell basics? (SWC maybe?)** - Yeah, we can do that I think

**Comment by makeclean** [comment]
**FLUKA or FLUKA source code? Is it necessary to be explicit about this like MCNP5? (honest question)**  - yes you have to apply for access to the executable, the point here is that these things block the installation if you don't have them

**Comment by makeclean** [comment]
**All volumes must belong to a group… will this cause FluDAG to fail or will the volumes be empty as in DAG-MCNP? In which case, can we just ask them to create an empty group. I think this is less confusing than assigning a pseudo-material to the graveyard** - MCNP5 used to assign unassigned volumes to vacuum, which is really quite dangerous (also will be remedied in a near future pr), hence the fact that we now require all volumes to belong to a group. The design decsion was the Vacuum and Graveyard are special, and they must exist as groups (if needed). Its also very convienient for checking, if not all volumes have property fail.

**Comment by makeclean** [comment]
**Do you need to add a volume to the MCNP implicit compliment material group as well? (Same as FLUDAG question)** - Only if you want to assign a material to the implicit compliment

**Comment by makeclean** [comment]
**Explain how we go about flooding a model maybe?** - Thats pretty standard MC practice

**Comment by makeclean** [comment]
**Some of this seems out of date. (e.g. mbconverting ACIS files to .h5m is no longer possible)** - I'm pretty sure that sections says trelis/cub files which we still do. Could you point me to a line number?

**Comment by pshriwise** [comment]
(Wish you could reply directly to comments :( )

**Some of this seems out of date. (e.g. mbconverting ACIS files to .h5m is no longer possible) - I'm pretty sure that sections says trelis/cub files which we still do. Could you point me to a line number?** 

_\- I'm pretty sure that sections says trelis/cub files which we still do. Could you point me to a line number?_

We may be able to for .cub files still, but I just tried this with a trelis file containing a single sphere w/ no luck.

File: doc/index.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ including MCNP5_, Geant4_, Fluka_, Tripoli4_, and Shift_. There are also efforts
 planned to integrate DAGMC into other codes such as MCNP6_, Serpent2_,
 OpenMC_, Phits_, and Frensie.
 
-DAGMC currently relies on using the solid modeling software Cubit_ or Trelis_ to
+DAGMC currently relies on using the solid modeling software Cubit_ (commercial name Trelis_) to
```
Comment (makeclean) [inline doc/index.rst]: Trelis is more than just a commerical version of Cubit right, doesnt it have more features for certain things?

File: doc/install/index.rst [hunk 2]
```diff
@@ -21,7 +21,7 @@ DAGMC installation. There are three main parts in the DAGMC installation:
     plugin
 
 Having completed the 3 stages of DAGMC installation, you will be ready to run
-DAGMC based radiation transport calculations. The
+DAGMC-based radiation transport calculations. The
```
Comment (makeclean) [inline doc/install/index.rst:24]: prefer no '-'

File: doc/usersguide/codes/fludag.rst [hunk 3]
```diff
@@ -147,13 +147,14 @@ Running FluDAG
...
-the subsequently defined geometry file, the user must produce the mat.inp file
+data from the CAD geometry we must first `facet the file <../trelis_basics.html#production-of-the-dagmc-geometry>`_
+using the Cubit/Trelis plugin. Using the subsequently defined geometry file, the user must
+produce the mat.inp file
 ::
 
     $ /path/to/fludag/executable/mainfludag geom.h5m
 
-Will load the named h5m file and produce the material assignments information.
+will load the named h5m file and produce the material assignments information.
```
Comment (makeclean) [inline doc/usersguide/codes/fludag.rst:157]: Why no capitalisation?

File: doc/usersguide/postprocessing.rst [hunk 4]
```diff
@@ -1,30 +1,30 @@
 Post-processing
 ===============
 One of the most	important aspects of a completed calculation is	being
-to view	any mesh based output alongside	geometry so as to best determine
+to view	any mesh-based output alongside	geometry so as to best determine
```
Comment (makeclean) [inline doc/usersguide/postprocessing.rst:4]: no hyphen

**Review by makeclean — COMMENTED** [review]
I think if you makes thse few changes, it'll be good to go

File: doc/usersguide/codes/fludag.rst (hunk 3 above)
Comment (pshriwise) [reply doc/usersguide/codes/fludag.rst:157]: because this looked like it was a continuation of the sentence above.

File: doc/index.rst (hunk 1 above)
Comment (pshriwise) [reply doc/index.rst]: That's possible. I just wanted to link them more closely than saying "Cubit or Trelis". Maybe something like "solid modeling software produces by csimsoft, Cubit or Trelis, ..."?

File: doc/install/index.rst (hunk 2 above)
Comment (pshriwise) [reply doc/install/index.rst:24]: truly? I feel like I see those two words hyphenated a lot...

**Comment by makeclean** [comment]
Awesome thanks :)

**Pull request merged**